x-pack/test/observability_ai_assistant_api_integration/tests/knowledge_base/knowledge_base.spec.ts

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { SUPPORTED_TRAINED_MODELS } from '../../../functional/services/ml/api';
+
+interface KnowledgeBaseEntry {
+  id: string;
+  text: string;
+}
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const ml = getService('ml');
+  const es = getService('es');
+  const TINY_ELSER = {
+    ...SUPPORTED_TRAINED_MODELS.TINY_ELSER,
+    id: SUPPORTED_TRAINED_MODELS.TINY_ELSER.name,
+  };
+  const KNOWLEDGE_BASE_API_URL = `/internal/observability_ai_assistant/kb`;
```

Comment:
we have the `observabilityAIAssistantAPIClient` service, which is a typed API client - can we use that one?